be/src/common/status.h

```diff
@@ -248,49 +256,42 @@ class Status {
 };
 
 // some generally useful macros
-#define RETURN_IF_ERROR(stmt) \
-    do { \
+#define RETURN_IF_ERROR(stmt)            \
+    do {                                 \
         const Status& _status_ = (stmt); \
-        if (UNLIKELY(!_status_.ok())) { \
-            return _status_; \
-        } \
-    } while (false)
-
-#define RETURN_IF_STATUS_ERROR(status, stmt) \
-    do { \
-        status = (stmt); \
-        if (UNLIKELY(!status.ok())) { \
-            return; \
-        } \
+        if (UNLIKELY(!_status_.ok())) {  \
+            return _status_;             \
+        }                                \
     } while (false)
 
-#define EXIT_IF_ERROR(stmt) \
-    do { \
-        const Status& _status_ = (stmt); \
-        if (UNLIKELY(!_status_.ok())) { \
-            string msg = _status_.get_error_msg(); \
-            LOG(ERROR) << msg;            \
-            exit(1); \
-        } \
+#define EXIT_IF_ERROR(stmt)                         \
```

Comment:
Emmm, why do this alignment? This will make it more difficult to maintain this

Reply:
This macro is basically unchanged, so there is no maintenance issue. 

Make `\` on the same column on the right, the code will be clearer, because readers **only need to focus on the implementation on the left** (same as the function). This macro is relatively small, and the advantages are not obvious.

On the other hand, putting `\` in the same column, in addition to being easy to read, also has another advantage, especially the macros that often change, **that it is easy to find errors**, because someone may forget to write `\`. In extreme cases, sometimes compile still will succeed even missing `\`.

Of course, as you said, there is no free lunch. The only problem with this alignment is that the number of diff lines may increase when the code is changed. But I think this "readability" and "easy to find errors" are more important.  :-)

Anyway, it is a style problem, if you don't agree, I can change it back.

Reply:
The maintain overhead is that if some one to change this code, he/she will realign these code. I think this is a matter of personal preference. Try to keep the original code unchanged.

Reply:
OK, change it back